packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.spec.ts

```diff
@@ -380,4 +378,122 @@ describe('AnchorTabs', () => {
             expect(document.activeElement).toBe(tab(0));
         });
     });
+
+    describe('scroll buttons', () => {
+        async function setup(): Promise<Fixture<AnchorTabs>> {
+            return await fixture<AnchorTabs>(
+                html`<nimble-anchor-tabs activeid="tab-two">
+                    <nimble-anchor-tab>Tab 1</nimble-anchor-tab>
+                    <nimble-anchor-tab id="tab-two">Tab 2</nimble-anchor-tab>
+                    <nimble-anchor-tab id="tab-three">Tab 3</nimble-anchor-tab>
+                    <nimble-anchor-tab id="tab-four">Tab 4</nimble-anchor-tab>
+                    <nimble-anchor-tab id="tab-five">Tab 5</nimble-anchor-tab>
+                    <nimble-anchor-tab id="tab-six">Tab 6</nimble-anchor-tab>
+                </nimble-anchor-tabs>`
+            );
+        }
+
+        let tabsPageObject: AnchorTabsPageObject;
+
+        beforeEach(async () => {
+            ({ element, connect, disconnect } = await setup());
+            await connect();
+            tabsPageObject = new AnchorTabsPageObject(element);
+        });
+
+        afterEach(async () => {
+            await disconnect();
+        });
+
+        it('should not show scroll buttons when the tabs fit within the container', () => {
+            expect(tabsPageObject.areScrollButtonsVisible()).toBeFalse();
+        });
+
+        it('should show scroll buttons when the tabs overflow the container', async () => {
+            await tabsPageObject.setTabsWidth(300);
+            await waitForUpdatesAsync(); // wait for the resize observer to fire
```

Comment:
Can we move the waitForUpdateAsync into the setTabsWidth page object? Looks like every instance of its call is followed by a manual waitForUpdatesAsync in the tests.

Also setTabsWidth already has a waitForUpdatesAsync call in it. So if a second one is required in there after width updates it's fine buut can document once there instead of in every usage below.